sorbet/rbi/gems/minitest@5.14.1.rbi

```diff
@@ -1,8 +1,15 @@
 # This file is autogenerated. Do not edit it by hand. Regenerate it with:
-#   tapioca sync
+#   tapioca generate minitest
```

Comment:
I needed to force tapioca to regenerate the RBI because when this RBI was first generated it `minitest-hooks` wasn't used in tapioca. `minitest-hooks` is bringing new modules from `minitest` that RBI didn't account for.

Reply:
TBH, I think we need a script to encapsulate how this is being done and use that as the generation command. This is good for now.

Reply:
I created an issue https://github.com/Shopify/sorbet-issues/issues/198